Formula/gnupg.rb

```diff
@@ -51,10 +51,9 @@ def install
     system "make", "install"
 
     on_macos do
-      (buildpath/"scdaemon.conf").write <<~EOS
+      (pkgetc/"scdaemon.conf").write <<~EOS
```

Comment:
I mislead you here. 🙂 Sorry!

Reply:
restored and squashed; though I did switch `etc/"gnupg"` to use `pkgetc`